static/app/cloudfoundry.js

```diff
@@ -76,6 +76,23 @@
                 });
         };
 
+        // Get quota usage data
+        this.getQuotaUsage = function(org) {
+            $http.get('/v2/organizations/' + org.guid + '/memory_usage')
```

Comment:
@msecret  :+1: , I'll definitely, keep this article around. Just refactored the code to make it a bit clearer and catch errors.